charts/vela-core/templates/defwithtemplate/gateway.yaml

```diff
@@ -125,6 +133,12 @@ spec:
 
         	// +usage=Specify a pathType for the ingress rules, defaults to "ImplementationSpecific"
         	pathType: *"ImplementationSpecific" | "Prefix" | "Exact"
+
```

Comment:
please refer to https://kubevela.net/zh/docs/contributor/cli-ref-doc, and update official doc